bpfd/src/command.rs

```diff
@@ -195,6 +199,10 @@ pub(crate) struct ProgramInfo {
     pub(crate) name: Option<String>,
     pub(crate) program_type: Option<u32>,
     pub(crate) location: Option<Location>,
+    pub(crate) global_data: Option<HashMap<String, Vec<u8>>>,
+    pub(crate) map_pin_path: Option<String>,
```

Comment:
Still TODO

Reply:
I studied this a little more. In the code, `BpfMap` is:
```
pub(crate) struct BpfMap {
    pub(crate) map_pin_path: String,
    pub(crate) used_by: Vec<Uuid>,
}
```
It is used in a hashmap that is indexed by the UUID of the Map owner. I can't really add `map_owner_uuid` to this struct because the instancing isn't correct. If you feel strongly about it, I can add `BpfMap` struct to the `ProgramInfo` struct and remove the two fields, `map_used_by` and `map_pin_path`, but not sure it buys us a lot in the end.